assets/src/edit-story/components/canvas/gridview/index.js

```diff
@@ -73,28 +91,67 @@ const Rectangle = styled.div`
   }
 `;
 
-function RangeControl({ value, onChange }) {
+// I think we can move this component to shared one in the future if we need this kind of spacer span.
```

Comment:
Remove comment or make this a more generic `@todo` comment — otherwise it is unclear who "I" refers to. I'd say remove it.

```suggestion
```